BE/Trusts/Trusts.rdf

```diff
@@ -117,8 +126,8 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>trust</rdfs:label>
-		<skos:definition>A fiduciary relationship in which one party, known as a trustor, gives another party, the trustee, the right to hold title to property or assets for the benefit of a third party, the beneficiary.</skos:definition>
-		<skos:editorialNote>This is a legal agreement between parties that someone owns, and is thereby an asset that they own. They can be taxed on this as any other asset. There are generally accepted things such as the source of funds that will determine who the revenue agency will go after. Definition reference URL: http://www.investopedia.com/terms/t/trust.asp</skos:editorialNote>
+		<skos:definition>fiduciary relationship and legal entity in which one party, known as a trustor, gives another party, the trustee, the right to hold title to and manage assets for the benefit of a third party, the beneficiary</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom>Barron&apos;s Dictionary of Finance and Investment Terms, Ninth Edition, 2014.</fibo-fnd-utl-av:adaptedFrom>
```

Comment:
I don't for the life of me see why we're adding this reference to a copyrighted work, which we know unsettles some users, when the definition here bears little to no resemblance to what's in the Barron's book (though TBH I'm looking at the 7th Ed).